src/main/deploy/states/release.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<!DOCTYPE statedata SYSTEM "stateData.dtd">
+<statedata>
+	<controlData identifier="openloop" 
+	             mode="POSITION_DEGREES"/>
+	<mechanismTarget stateIdentifier="SERVO_OPEN_OPEN"
+	                 controlDataIdentifier="openloop"
+                     controlDataIdentifier2="openloop"
+					 value="0.0"
+                     secondValue="0.0"/>
+
+	<mechanismTarget stateIdentifier="SERVO_OPEN_CLOSED"
+	                 controlDataIdentifier="openloop"
+                     controlDataIdentifier2="openloop"
+					 value="0.0"
+                     secondValue="0.0"/>
+
+	<mechanismTarget stateIdentifier="SERVO_CLOSED_OPEN"
+	                 controlDataIdentifier="openloop"
```

Comment:
value could be 0.0 for open, but should be something else for closed (e.g. 45.0); same with secondValue